docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -217,15 +223,29 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag
 <1> Configure the key's serializer.
 <2> Configure the value's serializer.
 
-=== Store-By-Value Caches with Copiers
+===== Implementation of a `Serializer`
+
+When implementing a `org.ehcache.spi.serialization.Serializer`, you obviously need to respect the interface contract.
+In addition, there are a few implicit contracts:
+
+* Your implementation must have a public constructor taking a `ClassLoader` as parameter.
+* There are two annotations that you have to use to mark your `Serializer` - a single implementation can use both:
```

Comment:
"have to use" or "may use"? The annotations are not mandatory when configured at the cache level, right? It is mandatory for the cache manager level config, though.